### DISCUSSION TITLE
Influence of lack of dielectric constants and Bron charges

### DISCUSSION THREAD

**Discussion by biglinn** [SSCHAcode/python-sscha#361] [Q&A] [answered]
Hi, @mesonepigreco:

When I do SSCHA dyn iterations `ensemble generation, calculation of energies and forces, free energy minimization`, I give trial dyns from QE ph.x.
So, I have a question. 

If trial dyns don't contain dielectric constants and Bron charges(such as, `smearing` for pw.x that requires `epsil = .false.` for ph.x), is there any influence?
Since the step `ensemble generation` will generate supercell, dose lack of dielectric constants and Bron charges lose **long-range infomation**?

**Comment by mesonepigreco** [comment, marked as answer]
Long-range effects given by effective charges and dielectric tensors are used only during the Fourier interpolation, which does not happen in a standard calculation. Indeed, to compute the spectral function, you need to perform the interpolation (tutorial 4: https://sscha.eu/Tutorials/tutorial_04_calculation_spectrum/), so in this case, the info about dielectric tensor and Born effective charges are used by the code.
